models/members.js

```diff
@@ -15,16 +15,32 @@ const fetchMembers = async () => {
   try {
     const snapshot = await userModel.where('isMember', '==', true).get()
 
-    const allMembers = []
+    const allMembers = { oldMembers: [], newMembers: [] }
 
     snapshot.forEach((doc) => {
-      allMembers.push({
+      const memberData = doc.data()
+      const curatedMemberData = {
         id: doc.id,
-        ...doc.data(),
+        ...memberData,
         tokens: undefined,
         phone: undefined,
         email: undefined
-      })
+      }
+      switch (memberData.userType) {
+        case 'new':
+          allMembers.newMembers.push(curatedMemberData)
+          break
+        case 'blocked':
+          // intentionally left blank for future use case
+          break
+        case 'member':
+          // intentionally left break statement to fall through default case
+          // so that doc with no userType field considered as members
```

Comment:
I get what is `new` and `blocked` are meant to do, but what is `member` doing? Didn't get that, can you please elaborate a little bit?

Reply:
TDLR; member case included to handle future scenario, not necessary right now.

IsMember is existing field. userType newly introduced field along with this PR. We need to discuss with AO, @sumitd94 
1. what to do with these two inter-related fields?
2. are we going to keep both the fields?
3. are we going to remove isMember field?
4. If we remove isMember, how to deal with related code changes?
5. If we remove isMemebr, how to deal with existing data?

Reply:
What do you suggest here?

Reply:
According to me

1. For now let's keep both, userType will contain the values like `new, blocked, member`. here the `member` is same as `isMember = true`
2. For now yes
3. In future yes
4. Need to think, open to suggestion
5. Need to think, open to suggestion

Reply:
I still didn't get why do we need `member` in `userType` if we already have `isMember` 😅

Reply:
@swarajpure, if we decide to keep isMember field forever then 'member' in 'userType' is not necessary. But that decision is not yet taken.

Reply:
@sumitd94 please unblock @rajakvk